src/MongoDB/ServerDescription.c

```diff
@@ -338,8 +330,6 @@ void php_phongo_serverdescription_init_ce(INIT_FUNC_ARGS) /* {{{ */
 	PHONGO_CE_FINAL(php_phongo_serverdescription_ce);
 	PHONGO_CE_DISABLE_SERIALIZATION(php_phongo_serverdescription_ce);
 
-	zend_class_implements(php_phongo_serverdescription_ce, 1, zend_ce_iterator);
```

Comment:
This caused the "cannot instantiate abstract class" errors: I don't think we need to implement the `Iterator` interface, but if we do we need to implement its methods.